### DISCUSSION TITLE
Any Hotline Miami fans here?

### DISCUSSION THREAD

**Discussion by Dr4gma** [space-wizards/space-station-14#16080] [Polls]
![HM](https://user-images.githubusercontent.com/107927438/236155574-81538ccf-bae6-4d95-96f2-d3a5121ebf21.png)

I made this outfit which is composed of
> Head 
> Jumpsuit

there is a side view but I didn't include it in the preview.

**Comment by PuroSlavKing** [comment]
the game already has a mask of Richard

**Reply by Dr4gma** [reply]
I never saw that lol. Thanks
